Description: Do not run the long test suite when
  building for debian.
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: yosys/Makefile
===================================================================
--- yosys.orig/Makefile	2014-11-08 22:46:03.066902046 +0100
+++ yosys/Makefile	2014-11-08 22:46:26.454901896 +0100
@@ -375,6 +375,6 @@
 -include kernel/*.d
 -include techlibs/*/*.d
 
-.PHONY: all top-all abc test install install-abc manual clean mrproper qtcreator
+.PHONY: all top-all abc install install-abc manual clean mrproper qtcreator
 .PHONY: config-clean config-clang config-gcc config-gcc-4.6 config-gprof config-sudo
 
